docs/source/tutorials/index.rst

```diff
@@ -143,6 +150,7 @@ your datasets and turn your good models into *great models*.
    :maxdepth: 1
    :hidden:
 
+   Natural language image search <semantic_image_search.ipynb>
```

Comment:
Can you rename this to something that will fit in one line in the sidebar? Maybe "Text-based image search"?